packages/mdc-list/_variables.scss

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 $mdc-list-divider-color-light: rgba(0, 0, 0, .12) !default;
-$mdc-list-divider-color-dark: rgba(255, 255, 255, .2) !default;
```

Comment:
I'm wondering one of two things here...

* Should we keep this variable and use `mdc-theme-contrast-tone` in our styles to determine whether to use light or dark based on the value of the background color?
* Should we remove this and also remove `-light` from the other variable?

cc @acdvorak

Reply:
So we would just keep this style in here that other people can reference if they create their own "dark-theme"?

Reply:
I vote for option 1: keep this variable and use `mdc-theme-contrast-tone`.

E.g.:

```sass
@at-root {
  $divider-color: if(
    mdc-theme-contrast-tone($mdc-theme-background) == "dark",
    $mdc-list-divider-color-dark,
    $mdc-list-divider-color-light
  );

  @include mdc-list-divider-color($divider-color);
}
```

That way, if the client overrides `$mdc-theme-background`, we will automatically set the correct divider color for them. If they don't like the color we choose, they can always override it with the mixin.

Reply:
Ok I see what you guys are saying now. I will update as stated here. Thanks!